x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/sample/SamplePayload.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.eql.execution.sample;
+
+import org.elasticsearch.core.TimeValue;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.xpack.eql.action.EqlSearchResponse.Event;
+import org.elasticsearch.xpack.eql.execution.payload.AbstractPayload;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.elasticsearch.xpack.eql.util.SearchHitUtils.qualifiedIndex;
+
+class SamplePayload extends AbstractPayload {
+
+    private final List<org.elasticsearch.xpack.eql.action.EqlSearchResponse.Sequence> values;
```

Comment:
Are values conceptually `Sequence`s? 
Structurally, for transport purposes, `Sequence` does the job, even though the events are supposed to be a Set (not a List).
Just wondering if it makes sense to keep the two concepts separate and have a specialized `EqlSearchResponse.Sample`